_topic_map.yml

```diff
@@ -60,8 +60,18 @@ Name: Getting started
 Dir: getting_started_rosa
 Distros: openshift-rosa
 Topics:
+- Name: Quickstart
```

Comment:
Is there a reason we are using "Quickstart" as one word and not two here and in `modules/rosa-quickstart-instructions.adoc`? Looks odd to me, but I find that we also have something in OCP called a "Quickstart template" (https://docs.openshift.com/container-platform/4.6/openshift_images/using-templates.html#templates-using-instant-app-quickstart_using-templates)

Reply:
No reason, I inherited that and thought it was our standard. And it was probably named that because of the template. Anyway, I have changed it to two words. Thanks!